### PR TITLE
Update e2e tests to work with keycloak

### PR DESCRIPTION
**Pull request by ewelinagr**

### CODE REVIEW

**Comment by ewelinagr** [comment]
Still have some tests failing, so fixing is still in progress

**Comment by ewelinagr** [comment]
All tests are green again!!!

File: transmart-data/test_data/keycloak/keycloak-test-resource.json [hunk 1]
```diff
@@ -0,0 +1,1542 @@
+{
+  "id" : "test",
```
Comment (forus) [inline transmart-data/test_data/keycloak/keycloak-test-resource.json]: what about having this realm settings as part of the keycloak libery test resources?

I will add this point to the following ticket's description https://jira.thehyve.nl/browse/TMT-261

**Comment by ewelinagr** [comment]
@forus Configuration of e2e is updated now. Tests pass for both transmart-api-server and transmart-server.

**Pull request merged**